ut/s3_common_utilities_test.cc

```diff
@@ -276,3 +276,12 @@ TEST_F(S3CommonUtilitiesTest, IsYamlValueNullTest6) {
   std::string input_string = "";
   EXPECT_FALSE(S3CommonUtilities::is_yaml_value_null(input_string));
 }
+
+TEST_F(S3CommonUtilitiesTest, VerifyXORObfuscation) {
+  std::shared_ptr<S3CommonUtilities::S3Obfuscator> pXORObfuscator =
+      std::make_shared<S3CommonUtilities::S3XORObfuscator>();
+  std::string input = "This is to be obfuscated. Check de-obfuscation";
+  std::string strObfuscated = pXORObfuscator->encode(input);
+  std::string output = pXORObfuscator->decode(strObfuscated);
+  EXPECT_TRUE(input == output);
```

Comment:
I am not sure if this suggestion would be better, but it we want to see to strings equal or not, we can use EXPECT_STREQ

Reply:
yes, can be done. But, at this point, will keep code unchanged.